src/views/shared/qr-scanner/qr-scanner.view.jsx

```diff
@@ -32,7 +32,7 @@ function QRScanner ({ hideMyCode, onSuccess, onError, onClose }) {
    * @returns {void}
    */
   function handleQRScan (result) {
-    if (result && Addresses.isHermezEthereumAddress(result)) {
+    if ((result && Addresses.isHermezEthereumAddress(result)) || (result && Addresses.isEthereumAddress(result))) {
```

Comment:
I would write the if as: `result && (isHermezEthereumAddress(result) || isEthereumAddress(result))`

Reply:
Done.